tests/tests.js

```diff
@@ -1,18 +1,124 @@
 const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const chai = require('chai')
```

Comment:
You should only import the chai library once. Once you've imported the full library so you can call the `use` function you can simply destructure the expect function off of that module.

```JavaScript
const chai = require('chai')
chai.use(sinonChai)
const { expect } = chai
```